lib/modules/manager/pep621/processors/uv.spec.ts

```diff
@@ -149,7 +86,20 @@ describe('modules/manager/pep621/processors/uv', () => {
         },
       },
     };
-    const dependencies = [{ depName: 'dep1' }, { depName: 'dep2' }];
+    const dependencies = [
+      {
+        depName: 'dep1',
+        packageName: 'dep1',
+      },
+      {
+        depName: 'dep2',
+        packageName: 'dep2',
+      },
+      {
+        depName: undefined,
+        packageName: undefined,
```

Comment:
In which scenario could this happen 🤔

Reply:
this should not happen

Reply:
Yeah, did that because of code coverage complaining that [this condition was not covered](https://github.com/renovatebot/renovate/blob/2e2e83b4d7316715e4627dc9c7be298f69d248a9/lib/modules/manager/pep621/processors/uv.ts#L49-L51), but maybe it's better to add `// istanbul ignore if` on the condition in that specific case.